### DISCUSSION TITLE
Removing or setting SMuFL glyph ID postfixes to a fixed string

### DISCUSSION THREAD

**Discussion by craigsapp** [rism-digital/verovio#3552] [Q&A]
Glyph IDs such as `E566` have an added random postfix, such as `srs021` in this case in the SVG output to verovio to avoid ID collisions when multiple verovio-generated svgs are placed in the same HTML document tree:

```xml
<symbol id="E566-srs021" viewBox="0 0 1000 1000" overflow="inherit">
```

Is there a way remove the random postfix, such as:

```xml
<symbol id="E566" viewBox="0 0 1000 1000" overflow="inherit">
```

or otherwise setting it to a user-selected string, such as

```xml
<symbol id="E566-my_postfix" viewBox="0 0 1000 1000" overflow="inherit">
```

**Comment by lpugin** [comment]
This can be added. I created a [feature request](https://github.com/rism-digital/verovio/issues/3592).
